pkg/storage/aws/s3.go

```diff
@@ -17,27 +18,25 @@ func BucketRegionIsDiscoverable(bucket string) bool {
 // GetBucketRegion returns the AWS region that a bucket is in, or an error
 // if the region cannot be determined.
 // copied from https://github.com/openshift/openshift-velero-plugin/pull/223/files#diff-da482ef606b3938b09ae46990a60eb0ad49ebfb4885eb1af327d90f215bf58b1
+// modified to aws-sdk-go-v2
 func GetBucketRegion(bucket string) (string, error) {
 	var region string
-
-	sessionInstance, err := session.NewSession()
+	// GetBucketRegion will attempt to get the region for a bucket using the client's configured region to determine
+	// which AWS partition to perform the query on.
+	// Client therefore needs to be configured with region.
+	// In local dev environments, you might have ~/.aws/config that could be loaded and set with default region.
+	// In cluster/CI environment, ~/.aws/config may not be configured, so set region explicitly.
+	// Also set to use anonymous credentials
+	cfg, err := config.LoadDefaultConfig(context.TODO(),
+		config.WithRegion("us-east-1"),
```

Comment:
Can you define:
```
const DefaultAwsBucketRegion = "us-east-1"
```

Reply:
This isn't default. This is the to specify a hinting server. See https://github.com/aws/aws-sdk-go-v2/pull/2716#:~:text=adding%20region%20hint%20in%20the%20example%20is%20good